voting/ethereum/instructions/validator_registration.md

```diff
@@ -0,0 +1,87 @@
+![](../instructions/enroll_validator_2.png)
+1. Click *Access*.
+* If the *Access* button appears disabled, make sure there are no trailing line feeds at the bottom of the *ABI / JSON Interface* text box.
+1. Select `register` in the drop down list *Read / Write Contract*
+![](../instructions/enroll_validator_3.png)
+The function parameters for the `register` call will appear in the form.
+1. Fill the fields labeled `name`, `ipAddress`, `website`, `OrbsAddress` 
+with your Validator's registration data.
+    - OrbsAddress must be formatted as a valid Ethereum address type.
+    - IP address must first be converted to a hexadecimal number representation:
+        - The hexadecimal number representation begins with `0x` followed by 8 hexadecimal digits, two for each byte in the IP address. for example: `216.58.207.46` will be formatted `0xd83acf2e`.
+        - Several online tools can perform the conversion. [Here is one example with screen shots](https://www.browserling.com/tools/ip-to-hex):<p>
+        ![](../instructions/IP_to_Hex_Converter_1.png)
+ 
+        Paste your Orbs node's IP address in the text box and click *Convert to Hex!* <p>
+        ![](../instructions/IP_to_Hex_Converter_2.png)
+        
+        Copy the hexadecimal number representation contained within the parentheses. Don't forget to include the prefix `0x`       
+![](../instructions/enroll_validator_4.png)
+
+1. Choose one of the options under *How would you like to access your wallet?*
+and provide your wallet information/credentials.
+In this example we choose to provide a mnemonic to open our wallet:
+![](../instructions/enroll_validator_5.png)
+Proceed to *Choose Address*. 
+1. Select your Validator's Ethereum address. 
+The address you choose will be the Validator's identification for voting later on.
+![](../instructions/enroll_validator_6.png)
+Make sure the account has a positive Ether balance for transaction fees. Then click *Unlock*.
+1. Adjust the *Gas Limit* as required. `500,000` should be more than enough (at the time this is being written a successful registration uses ~174000 Gas)
+![](../instructions/enroll_validator_7.png)
+Click *Write*, then *Sign Transaction* 
+![](../instructions/enroll_validator_8.png)
+Click *Send Transaction*
+![](../instructions/enroll_validator_9.png)
+Review, then click *Send*
+
+1. Once the transaction is sent, MyCrypto will provide a link to track the transaction status on Etherscan.
+Navigate to *Etherscan* by clicking *Verify (Etherscan)*
+![](../instructions/enroll_validator_10.png)
+
+1. Confirm the transaction has been accepted successfully.
+![](../instructions/etherscan_confirmation_1.png)
+ 
+Make sure you see 
+> TxReceipt Status:Success
+
+With sufficient block confirmations.
+
+[2]: https://ropsten.etherscan.io/address/0xd492757cee4c0e1159376aE7Da795fB6D949900a#code
+
+##### Notes
+* Registration values are unique among Validators, except for IP address which should be unique but is not enforced by the contract. 
```

Comment:
stress that: entering a name, website url, or orbs address which are already in use by another validator will revert and fail the transaction.

This was the purpose of the next line - but it appears the beginning of it was deleted and now it's broken

Reply:
fixed, why don't we enforce the IP? What about enforcement for Guardians?